salt/states/stormpath_account.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+'''
+Support for Stormpath
+
+.. versionadded:: Beryllium
+'''
+
```

Comment:
I'm not using the `print()` function here. Are we okay with `absolute_import` only?